lib/DebugInfo/Symbolize/BugsnagSymbolize.cpp

```diff
@@ -0,0 +1,100 @@
+#include "llvm/DebugInfo/Symbolize/BugsnagSymbolize.h"
+#include "llvm/DebugInfo/Symbolize/Symbolize.h"
+#include <cstdlib>
+#include <cstring>
+
+using namespace llvm;
+
+static void freeAndInvalidate(void** p) {
+  free((void*)*p);
```

Comment:
My understanding is that you should be calling "free" on memory that is allocated using "malloc" type operations and memory that is created using "new" should be deallocated using "delete", see: https://www.geeksforgeeks.org/delete-and-free-in-cpp/

Additionally, Is there a reason this is passing in a pointer to a pointer, then dereferencing it? Could this just not pass in (void* p) and call delete p.

Reply:
Simplified